sdks/python/apache_beam/runners/dataflow/dataflow_runner_test.py

```diff
@@ -259,18 +260,6 @@ def test_bigquery_read_streaming_fail(self):
                     PipelineOptions(self.default_properties)) as p:
         _ = p | beam.io.Read(beam.io.BigQuerySource('some.table'))
 
-  def test_biqquery_read_fn_api_fail(self):
```

Comment:
Wouldn't we still want an error most fo the time? (E.g. this is still broken on OSS runners.)